lib/sidekiq/logger.rb

```diff
@@ -115,8 +79,35 @@ def self.reopen_logs
       puts "Unable to reopen logs: #{ex.message}"
     end
 
-    def logger
-      Sidekiq::Logging.logger
+    class PrettyFormatter < ::Logger::Formatter
+      def call(severity, time, program_name, message)
+        "#{time.utc.iso8601(3)} #{::Process.pid} TID-#{Sidekiq.tid}#{format_context(Sidekiq.logger.context)} #{severity}: #{message}\n"
+      end
+
+      private
+
+      def format_context(context)
+        ' ' + context.join(' ') if context.any?
+      end
+    end
+
+    class WithoutTimestampFormatter < PrettyFormatter
+      def call(severity, time, program_name, message)
+        "#{::Process.pid} TID-#{Sidekiq.tid}#{format_context(Sidekiq.logger.context)} #{severity}: #{message}\n"
+      end
+    end
+
+    class JSONFormatter < PrettyFormatter
+      def call(severity, time, program_name, message)
+        Sidekiq.dump_json(
+          timestamp: time.utc.iso8601(3),
+          pid: ::Process.pid,
```

Comment:
I thought about `Sidekiq.pid` as an alias to `Process.pid`, but maybe it could confuse other contributors 🤔